.gitignore

```diff
@@ -1,4 +1,12 @@
+# Ignore a file actually called Makefile. The presence of this file interferes
+# with our Linux package building processes. Use the command:
+# make -f tasks.mak makefile_symlink
+# To make a local symlinked Makefile for development convenience.
 Makefile
 
 # Ignore generated credentials from google-github-actions/auth
 gha-creds-*.json
+
+# This folder is used to contain local builds of fluent-bit and otelopscol
+# for running transformation tests locally.
```

Comment:
Did a similar rewording in [a4b98df](https://github.com/GoogleCloudPlatform/ops-agent/pull/1722/commits/a4b98df8ab532ea8364860adeae69072f3d3b376)

Still opted to use the `used to` verbiage in some way, since it only contains those local builds when running integration tests.